docs/operate/customize/dois.md

```diff
@@ -69,6 +69,21 @@ DATACITE_FORMAT = "{prefix}/inveniordm.{id}"
     [Cool DOIs](https://doi.org/10.5438/55e5-t5c0) and why it might not be a
     good idea.
 
+#### OAI-PMH
+
+The OAI-PMH server's metadata format ``oai_datacite`` that allows you to harvest record from InvenioRDM in DataCite XML needs to be configured with your DataCite data center symbol. This is only required if you want your records to be harvestable in DataCite XML format.
+
+```python
+DATACITE_DATACENTER_SYMBOL = "CERN.INVENIORDM"
+```
+
+### Versioning and externally managed DOI
+
+By default, InvenioRDM allows versioning for any DOI type - internally or externally managed. Internally managed DOI is a DOI which is given thanks to InvenioRDM feature which allows us to configure the DOI registration on DataCite (check #Enable DOI registration). The external DOIs are not minted by our instance and in some cases repository manager decides to disallow versioning of records identified by external DOI. To disable versioning for external DOIs you need to set:
```

Comment:
Small improvements to text while we are here:

```suggestion
By default, InvenioRDM allows versioning for any DOI type - internally or externally managed. Internally managed DOIs are DOIs registered by your instance and minted on Datacite (enabled by `DATACITE_ENABLED = True`). External DOIs are DOIs already minted and simply passed to your instance to be registered. As such, internally managing the next version of a record with an external DOI may not be desired. To disable versioning for external DOIs you need to set:
```

Reply:
The DATACITE_ENABLED config does the entire DOI setup....which should probably be changed at some point. I've reworded your suggestion a bit.